engine-src/Game/LambdaHack/Client/UI/FrameM.hs

```diff
@@ -102,14 +102,14 @@ promptGetKey dm ovs onBlank frontKeyKeys = do
   keyPressed <- anyKeyPressed
   report <- getsSession $ newReport . shistory
   let msgDisturbs = anyInReport disturbsResting report
-  InGameMacro lastPlayOld <- getsSession slastPlay
-  km <- case lastPlayOld of
+  lastPlayOld <- getsSession slastPlay
+  km <- case unMacro lastPlayOld of
     km : kms | not keyPressed
```

Comment:
What I meant is `InGameMacro (km : kms)` here instead of the destructor, but it's a petty detail, of course.

Reply:
Oh, sorry! I completely misunderstood you. You're right, it's much more readable.